src/GraphQL/GraphQL.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\..\packages\Antlr4.4.5-alpha002\build\Antlr4.props" Condition="Exists('..\..\packages\Antlr4.4.5-alpha002\build\Antlr4.props')" />
+  <Import Project="C:\projects\alex\Platform\trunk\Nuget\Repository\Antlr4.4.5-alpha002\build\Antlr4.props" Condition="Exists('C:\projects\alex\Platform\trunk\Nuget\Repository\Antlr4.4.5-alpha002\build\Antlr4.props')" />
```

Comment:
Yeah.. I was a bit quick with my `commit -am` command :)  Will clean this up now